### PR TITLE
Added comments for generic timer

### PR DESCRIPTION
**Pull request by HyeJi**
Register generic timer irqs

### CODE REVIEW

**Comment by kesl** [comment]
Can one of the admins verify this patch?

File: hypervisor/generic_timer/generic_timer.h [hunk 1]
```diff
@@ -30,7 +30,15 @@ enum {
     GENERIC_TIMER_REG_VIRT_OFF,
 };
 
-/* Calling this function is required only once in the entire system. */
+/**
+* @brief Register generic timer irqs such as HYP, NSP
```
Comment (Wonseok) [inline hypervisor/generic_timer/generic_timer.h]: s/Register generic timer irqs such as HYP, NSP/Register generic timer irqs such as HYP, NSP and VIR/

File: hypervisor/generic_timer/generic_timer.h [hunk 2]
```diff
@@ -30,7 +30,15 @@ enum {
...
+*
+* HYP : 26
```
Comment (Wonseok) [inline hypervisor/generic_timer/generic_timer.h]: DEVICE          : IRQ number
HYP_TIMER   : 26
NSP_TIMER   : 27
VIR_TIMER   : 30

File: hypervisor/generic_timer/generic_timer.h [hunk 3]
```diff
@@ -30,7 +30,15 @@ enum {
...
+* NSP : 27
+* VIR : 30
+*
+* @return Returns success only.
```
Comment (Wonseok) [inline hypervisor/generic_timer/generic_timer.h]: Use MACRO instead of SUCCESS.

File: hypervisor/generic_timer/generic_timer.c [hunk 4]
```diff
@@ -130,11 +130,19 @@ static inline uint64_t generic_timer_reg_read64(int reg)
     return val;
 }
 
+/**
+ * @brief Each interrup source is identified by a unique ID.
```
Comment (Wonseok) [inline hypervisor/generic_timer/generic_timer.c]: s/interrup/interrupt

**Pull request closed without merging**